src/datasources/asset-utilization/helper.ts

```diff
@@ -0,0 +1,311 @@
+import {
+    AssetUtilizationHistory,
+    Interval,
+    IntervalWithHeartbeat,
+    IsPeak, SystemLinkError,
+    UtilizationTimeFrequency,
+    Weekday
+} from "./types";
+
+export const minuteInSeconds = 60 * 1000;
```

Comment:
This was already defined in constants.ts

Reply:
removed